thorlcr/graph/thgraphslave.hpp

```diff
@@ -474,6 +474,7 @@ class graphslave_decl CJobSlave : public CJobBase
     Owned<IPropertyTree> workUnitInfo;
     size32_t oldNodeCacheMem;
     unsigned channelMemoryMB;
+    unsigned actInitWaitTimeMins = 0;
```

Comment:
Doesn't really matter, but why not initialize this to DEFAULT_MAX_ACTINITWAITTIME ?

Reply:
could do, I'll change.

Reply:
@mckellyln - NB: also changed this to DEFAULT_MAX_ACTINITWAITTIME_MINS and corrected constant as mentioned here: https://github.com/hpcc-systems/HPCC-Platform/pull/11903#discussion_r235661190